frontend/__tests__/Pages/Register.test.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { render, wait, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import fetch from 'jest-fetch-mock';
+
+import App from '../../src/App';
+
+global.fetch = fetch;
+
+describe('Template component', () => {
+  // mock store and history
+  let store;
+
+  beforeEach(() => {
+    // Set up mock state store
+    const mockStore = configureStore([thunk]);
+    // Initialize mockstore with empty state
+    const initialEmptyState = {};
+    store = mockStore(initialEmptyState);
+  });
+
+  it('should render the dropdown component on width < 600px', async () => {
```

Comment:
Where is 600px set here? How big is the window it is being "rendered" to?

Reply:
The title is wrong after i forogot to change it after copy paste. I will update the tests